roles/zabbix_server/tasks/initialize-pgsql.yml

```diff
@@ -131,4 +131,4 @@
       vars:
         is_legacy_version: "{{ zabbix_server_version is version('7.0', '<') }}"
         legacy_path: /usr/share/zabbix-sql-scripts/postgresql/timescaledb.sql
```

Comment:
Oh crap, they changed the path for timescale 3 times in the last 3 releases :sweat: